### PR TITLE
Azerbaijani locale file

### PR DESCRIPTION
**Pull request by orkhan**

### CODE REVIEW

File: config/locales/az.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      resend_confirmation_instructions:
+        title: "Выслать повторно письмо с активацией"
+        submit: "Выслать повторно письмо с активацией"
+      links:
+        sign_up: "Qeydiyyat"
+        sign_in: "Giriş"
+        forgot_your_password: "Şifrəni unutmusunuz?"
+        sign_in_with_omniauth_provider: "%{provider} vasitəsilə daxil ol"
+        resend_unlock_instructions: "Повторная отправка инструкций разблокировки"
+        resend_confirmation_instructions: "Повторная отправка инструкций подтверждения"
```
Comment (Fivell) [inline config/locales/az.yml]: @orkhan , why part of localisation  strings are russian ?

**Comment by seanfcarroll** [comment]
@orkhan are you able to respond to the Russian question? If this is going to go ahead, I wonder if you would consider localising Ransack also, as it is used by ActiveAdmin. I have opened an issue [here](https://github.com/activerecord-hackery/ransack/issues/905).

**Comment by deivid-rodriguez** [comment]
I rebased this PR and added a changelog entry. It should be good to go now.

Only concern is whether https://github.com/activeadmin/activeadmin/pull/5078/commits/93cd894da9fc496edcf8ef6c9f23bca7222e595d makes sense at all. I added it so that the `bin/i18n-tasks check-consistent-interpolations` would pass.

**Comment by deivid-rodriguez** [comment]
I think we can go ahead a merge this. If there's a problem with my changes, Azerbaijani speaking users can find it out after the next release :). Settings this as ready.

**Comment by deivid-rodriguez** [comment]
@javierjulio I'm going to merge this and release 2.5.0, if that sounds good to you.

**Comment by javierjulio** [comment]
Oh wow sorry not sure how I missed this. Yes I think that’s a great idea. I’ll approve so you can merge. Thanks!

**Pull request merged**